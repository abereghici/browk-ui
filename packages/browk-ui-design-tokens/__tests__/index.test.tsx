import theo from 'theo';
import path from 'path';

beforeAll(() => {
  theo.registerTransform('web', ['color/rgb']);
});

describe('Design Tokens', () => {
  it('matches the Global theme', () => {
    const theme = theo.convertSync({
      transform: {
        type: 'web',
        file: path.resolve(__dirname, '../tokens/tokens.yml'),
      },
      format: {
        type: 'json',
      },
    });
    expect(theme).toMatchSnapshot();
  });
});
