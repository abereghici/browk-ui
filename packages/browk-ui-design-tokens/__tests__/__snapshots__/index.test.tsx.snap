// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Design Tokens matches the Global theme 1`] = `
"{
  \\"color-background\\": \\"rgb(255, 255, 255)\\",
  \\"color-background-strong\\": \\"rgb(250, 250, 250)\\",
  \\"color-background-brand\\": \\"rgb(110, 65, 226)\\",
  \\"color-background-brand-strong\\": \\"rgb(17, 17, 17)\\",
  \\"color-background-overlay\\": \\"rgba(0, 0, 0, 0.5)\\",
  \\"color-border\\": \\"rgb(241, 241, 241)\\",
  \\"border-radius-0\\": \\"0\\",
  \\"border-radius-10\\": \\"2px\\",
  \\"border-radius-20\\": \\"4px\\",
  \\"border-radius-30\\": \\"8px\\",
  \\"border-radius-circle\\": \\"50%\\",
  \\"border-width-0\\": \\"0\\",
  \\"border-width-10\\": \\"1px\\",
  \\"border-width-20\\": \\"2px\\",
  \\"border-width-30\\": \\"4px\\",
  \\"border-width-40\\": \\"8px\\",
  \\"shadow\\": \\"0 4px 4px rgba(51,51,51, 0.04), 0 4px 16px rgba(51,51,51, 0.08)\\",
  \\"shadow-active\\": \\"0 4px 4px rgba(51,51,51, 0.04), 0 4px 56px rgba(51,51,51, 0.16)\\",
  \\"shadow-hover\\": \\"0 4px 4px rgba(51,51,51, 0.04), 0 4px 24px rgba(51,51,51, 0.24)\\",
  \\"shadow-success\\": \\"0 4px 4px rgba(39, 174, 96, 0.04), 0 4px 24px rgba(39, 174, 96, 0.24)\\",
  \\"shadow-error\\": \\"0 4px 4px rgba(255, 196, 196, 0.04), 0 4px 24px rgba(255, 196, 196, 0.24)\\",
  \\"font-family-text\\": \\"Roboto, -apple-system, Ubuntu, sans-serif\\",
  \\"font-family-code\\": \\"Montserrat, Courier, monospace\\",
  \\"font-size-10\\": \\"0.813rem\\",
  \\"font-size-20\\": \\"1rem\\",
  \\"font-size-30\\": \\"1.25rem\\",
  \\"font-size-40\\": \\"1.5rem\\",
  \\"font-size-50\\": \\"2rem\\",
  \\"font-size-60\\": \\"2.5rem\\",
  \\"font-size-70\\": \\"3rem\\",
  \\"font-size-80\\": \\"3.5rem\\",
  \\"font-weight-light\\": \\"400\\",
  \\"font-weight-normal\\": \\"400\\",
  \\"font-weight-medium\\": \\"500\\",
  \\"font-weight-semibold\\": \\"600\\",
  \\"font-weight-bold\\": \\"700\\",
  \\"line-height-0\\": \\"0\\",
  \\"line-height-80\\": \\"4.55rem\\",
  \\"line-height-70\\": \\"4rem\\",
  \\"line-height-60\\": \\"3.25rem\\",
  \\"line-height-50\\": \\"2.75rem\\",
  \\"line-height-40\\": \\"2rem\\",
  \\"line-height-30\\": \\"1.75rem\\",
  \\"line-height-20\\": \\"1.5rem\\",
  \\"line-height-10\\": \\"1.25rem\\",
  \\"color-brand-highlight\\": \\"rgb(255, 217, 18)\\",
  \\"color-gray-0\\": \\"rgb(255, 255, 255)\\",
  \\"color-gray-70\\": \\"rgb(17, 17, 17)\\",
  \\"color-gray-60\\": \\"rgb(51, 51, 51)\\",
  \\"color-gray-50\\": \\"rgb(107, 107, 107)\\",
  \\"color-gray-40\\": \\"rgba(17, 17, 17, 0.48)\\",
  \\"color-brand\\": \\"rgb(110, 65, 226)\\",
  \\"color-gray-30\\": \\"rgb(212, 212, 212)\\",
  \\"color-gray-20\\": \\"rgb(241, 241, 241)\\",
  \\"color-gray-10\\": \\"rgb(250, 250, 250)\\",
  \\"size-icon-80\\": \\"4.55rem\\",
  \\"size-icon-70\\": \\"4rem\\",
  \\"size-130\\": \\"74rem\\",
  \\"size-icon-60\\": \\"3.25rem\\",
  \\"size-90\\": \\"57.5rem\\",
  \\"size-120\\": \\"74rem\\",
  \\"size-icon-50\\": \\"2.75rem\\",
  \\"size-80\\": \\"51rem\\",
  \\"size-0\\": \\"0\\",
  \\"size-110\\": \\"70.5rem\\",
  \\"size-icon-40\\": \\"2rem\\",
  \\"size-70\\": \\"44.5rem\\",
  \\"size-100\\": \\"64rem\\",
  \\"size-icon-30\\": \\"1.75rem\\",
  \\"size-60\\": \\"38rem\\",
  \\"size-icon-20\\": \\"1.5rem\\",
  \\"size-icon-10\\": \\"1.25rem\\",
  \\"size-50\\": \\"31.5rem\\",
  \\"size-40\\": \\"25rem\\",
  \\"size-30\\": \\"18.5rem\\",
  \\"size-20\\": \\"12rem\\",
  \\"size-10\\": \\"5.5rem\\",
  \\"space-70\\": \\"1.5rem\\",
  \\"space-negative-100\\": \\"-2.25rem\\",
  \\"space-60\\": \\"1.25rem\\",
  \\"space-negative-200\\": \\"-4.75rem\\",
  \\"space-190\\": \\"4.5rem\\",
  \\"space-50\\": \\"1rem\\",
  \\"space-180\\": \\"4.25rem\\",
  \\"space-40\\": \\"0.75rem\\",
  \\"space-170\\": \\"4rem\\",
  \\"space-30\\": \\"0.5rem\\",
  \\"space-160\\": \\"3.75rem\\",
  \\"space-20\\": \\"0.25rem\\",
  \\"space-150\\": \\"3.5rem\\",
  \\"space-10\\": \\"0.125rem\\",
  \\"space-140\\": \\"3.25rem\\",
  \\"space-130\\": \\"3rem\\",
  \\"space-120\\": \\"2.75rem\\",
  \\"space-negative-90\\": \\"-2rem\\",
  \\"space-110\\": \\"2.5rem\\",
  \\"space-0\\": \\"0\\",
  \\"space-negative-80\\": \\"-1.75rem\\",
  \\"space-100\\": \\"2.25rem\\",
  \\"space-negative-70\\": \\"-1.5rem\\",
  \\"space-200\\": \\"4.75rem\\",
  \\"space-negative-60\\": \\"-1.25rem\\",
  \\"space-negative-50\\": \\"-1rem\\",
  \\"space-negative-40\\": \\"-0.75rem\\",
  \\"space-negative-30\\": \\"-0.5rem\\",
  \\"space-negative-20\\": \\"-0.25rem\\",
  \\"space-negative-10\\": \\"-0.125rem\\",
  \\"space-negative-190\\": \\"-4.5rem\\",
  \\"space-negative-180\\": \\"-4.25rem\\",
  \\"space-negative-170\\": \\"-4rem\\",
  \\"space-negative-160\\": \\"-3.75rem\\",
  \\"space-negative-150\\": \\"-3.5rem\\",
  \\"space-negative-140\\": \\"-3.25rem\\",
  \\"space-negative-130\\": \\"-3rem\\",
  \\"space-90\\": \\"2rem\\",
  \\"space-negative-120\\": \\"-2.75rem\\",
  \\"space-80\\": \\"1.75rem\\",
  \\"space-negative-110\\": \\"-2.5rem\\",
  \\"color-text\\": \\"rgb(0, 0, 0)\\",
  \\"color-text-inverse\\": \\"rgb(255, 255, 255)\\",
  \\"color-text-link\\": \\"rgb(242, 202, 202)\\",
  \\"color-text-icon\\": \\"rgb(51, 51, 51)\\",
  \\"z-index-0\\": \\"0\\",
  \\"z-index-90\\": \\"90\\",
  \\"z-index-80\\": \\"80\\",
  \\"z-index-70\\": \\"70\\",
  \\"z-index-60\\": \\"60\\",
  \\"z-index-50\\": \\"50\\",
  \\"z-index-40\\": \\"40\\",
  \\"z-index-30\\": \\"30\\",
  \\"z-index-20\\": \\"20\\",
  \\"z-index-10\\": \\"10\\"
}"
`;
